Open_Source_Events/KWOC/README.md

```diff
@@ -0,0 +1,12 @@
+<h1 align="center"> Kharagpur Winter of Code 2020</h1>
+
+<h2> About KWOC'20 </h2>
+
+Kharagpur Winter of Code is a 5-week long online program for students who are new to open source software development. 
+The program not only helps students to get involved in open source, but also prepares them for many open source summer programs; Google Summer of Code being one of them.
+
+<p align="center">
+  <a href="https://kwoc.kossiitkgp.org/">
+    <img src="https://github.com/Vishal-raj-1/Rotten-Scripts/blob/winter-of-code/Open_Source_Events/KWOC/kwoc.png" alt="Logo" />
```

Comment:
Change this. If you delete your Fork, the whole Image would be lost. Instead, upload the Image on Imgur and share the relative link. This will ensure that the Image would be part of the Project.

Reply:
I have uploading the image. let me give the relative path for that ./kwoc.png. It will work.